openlane_run/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Run design with latest OpenLane release
+
+inputs:
+  design_name:
+    description: Name of design to run
+    required: true
+  pdk_root:
+    description: PDK root path
+    required: true
+  ol_tag:
+    description: OpenLane tag
+    default: latest
+
+
+runs:
+  using: composite
+  steps:
+
+    - name: Export PDK ROOT
+      shell: bash
+      run: |
+        echo "PDK_ROOT="${{ inputs.pdk_root }} >> $GITHUB_ENV
+        echo "OPENLANE_IMAGE_NAME=efabless/openlane:${{ inputs.ol_tag }}" >> $GITHUB_ENV
+
+    - name: Build OpenLane docker image
```

Comment:
Does it actually build the OpenLane docker image or just fetch it?

Reply:
Pull or build if not available.